lib/ap/plan9/rename.c

```diff
@@ -43,7 +43,7 @@ rename(const char *from, const char *to)
 	f = f? f+1 : (char *)from;
 	t = t? t+1 : (char *)to;
 	n = 0;
-	if(f-from==t-to && strncmp(from, to, f-from)==0){
+	if(0 && (f-from==t-to && strncmp(from, to, f-from)==0)){
```

Comment:
I said that: "Fixed rename(), which has a check through _dirwstat that wasn't working." Reasons about why is not working seems being related with permissions for APEX over the FS, what is causing that _dirwstat always was returning -1. But this is a theory, this should be tested with a properly FS with auth enabled.